### ISSUE TITLE
Dungeon collapses if a worn amulet turns into an amulet of change

### ISSUE THREAD

**Issue by AndrioCelos** [NetHack/NetHack#106]
As far as I'm aware, this can only happen if you dip the amulet into a potion of polymorph:
```
What do you want to dip a concave amulet (being worn) into? [fgho or ?*] o
You are suddenly very feminine!  The amulet disintegrates!
Oops...

Suddenly, the dungeon collapses.

extract_nobj: object lost
zsh: abort (core dumped)  nethack -D
```
Loading the save file reveals that I am still female, and the concave amulet is still in my inventory but no longer worn.
```
Unix NetHack Version 3.6.1 - last build Mon May  7 00:45:44 2018.
```

**Comment by dwntwn512** [comment]
In around 20 years of playing nethack I have only had this happen once. I was reading a post on a forum about how a player had attacked Nefet by mistake and gotten blocked out of the home quest.  In the past I had killed the apprentices but could not recall if I had attacked or killed Nefet. I used to use the actual zip folder and play on my pc., but recently switchedto the version in the app store because it has larger  easy to read fonts. I know you can change the screen size on the downloadable version but the sizes are not that appealing and do not really increase the font sizes so that it is easy to read the screen. As this was done on the app from the app store I do not know if this will work on all versions. This was done with a save file so not to lose my game. I already had the eye and all other quest completed and was just playing killing enemies collecting potions to poly flip and building up my hoard  of items.

The apprentices can be killed as long as they are not in Nefets line of sight.  If he see's you killing them he kicks you out of the quest. So if you want the items the apprentices are carrying open the various doors and wait for them to leave the throne room. You also can polymorph them. For the most part this works fine but if they turn into a  monster that is normally friendly killing them will upset Nefet but I may have been in the throne room and he possible could have been watching when he got mad. When I went to kill Nefet, 1st I tried attacking Nefet  but he just got mad and kicked me out of the quest and sent me back to the normal dungeon. I then tried  FOD and found it kills him right away so you do not get kicked out of the quest. However the eye would no longer work and in fact I could not even hold the eye. I then tried poly morphing Nefet and found if killed after being polymorph the eye was again made useless .  After being poly morphed  Nefet comes after you and if not killed with FOD if he starts to fight you you will get kicked out of the quest.
earlier I noticed  if creatures hit the magic port to go to Ladios or the quest they would be found dead on the other end of the portal. So I decided to give this a try to see if he can be killed by something other than me. It took a couple of tries. I polymorphed Nefet from a distacne. Then as he started to come after me I hit him with the teleport away spell. Eventually i got him teleported outside of the castle and he was above the outside of the castle so I left and made my way to the level portal. I stood with the portal between me and Nefet. Eventually I was able to get him to step on the portal when trying to get to me and he disappeared. I had no way of knowing if he would be on the other side alive or dead. I used the eye to port to Ladios then went through the dungeon till I was at the level the port to the quest is at. As I approached the portal there was a corpse on the portal. I walked up and touched the portal and saw a message that Nefet dead body was on the port. Then the message saying the dungeon had collapsed around me and was directed to send the error in to be looked at. I only did this 1 time and do not know if this would have happened if I waited for the corpse to decay and disappear. I just assumed with  the way Nefet died the game was in a error state and had to be ended. I just loaded my saved game and went back to normal play. I do not know if the collapse was from touching the corpse of Nefet, or from touching the portal as the quest level should be blocked due to attacking and causing Nefets death.

Also at a later time I thought of using a helm of opposite alignment when killing Nefet to see if I could kill him with out upsetting my god and perhaps keep the use of the eye. I never got around to trying this as there is really nothing to gain from killing Nefet as he has no items unlike his apprentices. and you lose the ability to use the eye and can no longer port to parts of the dungeon. Still causing the dungeon to collapse and crash the game was kind of cool but totally a useless waste of time.

**Comment by dwntwn512** [comment]
Skip to content
Search or jump to…

Pull requests
Issues
Marketplace
Explore
 
@dwntwn512 
Learn Git and GitHub without any code!
Using the Hello World guide, you’ll start a branch, write comments, and open a pull request.


NetHack
/
NetHack
105
1.3k279
Code
Issues
41
Pull requests
49
Actions
Projects
Wiki
Security
9
Insights
Dungeon collapses if a worn amulet turns into an amulet of change #106
 Closed	AndrioCelos opened this issue on Jun 17, 2018 · 1 comment
 Closed
Dungeon collapses if a worn amulet turns into an amulet of change
#106
AndrioCelos opened this issue on Jun 17, 2018 · 1 comment
Comments
@AndrioCelos
 
AndrioCelos commented on Jun 17, 2018
As far as I'm aware, this can only happen if you dip the amulet into a potion of polymorph:

What do you want to dip a concave amulet (being worn) into? [fgho or ?*] o
You are suddenly very feminine!  The amulet disintegrates!
Oops...

Suddenly, the dungeon collapses.

extract_nobj: object lost
zsh: abort (core dumped)  nethack -D
Loading the save file reveals that I am still female, and the concave amulet is still in my inventory but no longer worn.

Unix NetHack Version 3.6.1 - last build Mon May  7 00:45:44 2018.
@nhcopier nhcopier closed this in 4bce58f on Jun 17, 2018
nhcopier pushed a commit that referenced this issue on Jun 18, 2018
PatR
more polymorph of worn item …
1822f54
@dwntwn512
 
dwntwn512 commented now
In around 20 years of playing nethack I have only had this happen once. I was reading a post on a forum about how a player had attacked Nefet by mistake and gotten blocked out of the home quest. In the past I had killed the apprentices but could not recall if I had attacked or killed Nefet. I used to use the actual zip folder and play on my pc., but recently switchedto the version in the app store because it has larger easy to read fonts. I know you can change the screen size on the downloadable version but the sizes are not that appealing and do not really increase the font sizes so that it is easy to read the screen. As this was done on the app from the app store I do not know if this will work on all versions. This was done with a save file so not to lose my game. I already had the eye and all other quest completed and was just playing killing enemies collecting potions to poly flip and building up my hoard of items.

The apprentices can be killed as long as they are not in Nefets line of sight. If he see's you killing them he kicks you out of the quest. So if you want the items the apprentices are carrying open the various doors and wait for them to leave the throne room. You also can polymorph them. For the most part this works fine but if they turn into a monster that is normally friendly killing them will upset Nefet but I may have been in the throne room and he possible could have been watching when he got mad. When I went to kill Nefet, 1st I tried attacking Nefet but he just got mad and kicked me out of the quest and sent me back to the normal dungeon. I then tried FOD and found it kills him right away so you do not get kicked out of the quest. However the eye would no longer work and in fact I could not even hold the eye. I then tried poly morphing Nefet and found if killed after being polymorph the eye was again made useless . After being poly morphed Nefet comes after you and if not killed with FOD if he starts to fight you you will get kicked out of the quest.
earlier I noticed if creatures hit the magic port to go to Ladios or the quest they would be found dead on the other end of the portal. So I decided to give this a try to see if he can be killed by something other than me. It took a couple of tries. I polymorphed Nefet from a distacne. Then as he started to come after me I hit him with the teleport away spell. Eventually i got him teleported outside of the castle and he was above the outside of the castle so I left and made my way to the level portal. I stood with the portal between me and Nefet. Eventually I was able to get him to step on the portal when trying to get to me and he disappeared. I had no way of knowing if he would be on the other side alive or dead. I used the eye to port to Ladios then went through the dungeon till I was at the level the port to the quest is at. As I approached the portal there was a corpse on the portal. I walked up and touched the portal and saw a message that Nefet dead body was on the port. Then the message saying the dungeon had collapsed around me and was directed to send the error in to be looked at. I only did this 1 time and do not know if this would have happened if I waited for the corpse to decay and disappear. I just assumed with the way Nefet died the game was in a error state and had to be ended. I just loaded my saved game and went back to normal play. I do not know if the collapse was from touching the corpse of Nefet, or from touching the portal as the quest level should be blocked due to attacking and causing Nefets death.

Also at a later time I thought of using a helm of opposite alignment when killing Nefet to see if I could kill him with out upsetting my god and perhaps keep the use of the eye. I never got around to trying this as there is really nothing to gain from killing Nefet as he has no items unlike his apprentices. and you lose the ability to use the eye and can no longer port to parts of the dungeon. Still causing the dungeon to collapse and crash the game was kind of cool but totally a useless waste of time.

@dwntwn512

 
 
Leave a comment

Attach files by dragging & dropping, selecting or pasting them.
Remember, contributions to this repository should follow our GitHub Community Guidelines.
Assignees
No one assigned
Labels
None yet
Projects
None yet
Milestone
No milestone
Linked pull requests
Successfully merging a pull request may close this issue.

None yet
Notifications
Customize
You’re receiving notifications because you commented.
2 participants
@AndrioCelos
@dwntwn512
© 2020 GitHub, Inc.
Terms
Privacy
Security
Status
Help
Contact GitHub
Pricing
API
Training
Blog
About
dwntwn512 commented now In around 20 years of playing nethack I have only had this happen once. I was reading a post on a forum about how a player had attacked Nefet by mistake and gotten blocked out of the home quest. In the past I had killed the apprentices but could not recall if I had attacked or killed Nefet. I used to use the actual zip folder and play on my pc., but recently switchedto the version in the app store because it has larger easy to read fonts. I know you can change the screen size on the downloadable version but the sizes are not that appealing and do not really increase the font sizes so that it is easy to read the screen. As this was done on the app from the app store I do not know if this will work on all versions. This was done with a save file so not to lose my game. I already had the eye and all other quest completed and was just playing killing enemies collecting potions to poly flip and building up my hoard of items. The apprentices can be killed as long as they are not in Nefets line of sight. If he see's you killing them he kicks you out of the quest. So if you want the items the apprentices are carrying open the various doors and wait for them to leave the throne room. You also can polymorph them. For the most part this works fine but if they turn into a monster that is normally friendly killing them will upset Nefet but I may have been in the throne room and he possible could have been watching when he got mad. When I went to kill Nefet, 1st I tried attacking Nefet but he just got mad and kicked me out of the quest and sent me back to the normal dungeon. I then tried FOD and found it kills him right away so you do not get kicked out of the quest. However the eye would no longer work and in fact I could not even hold the eye. I then tried poly morphing Nefet and found if killed after being polymorph the eye was again made useless . After being poly morphed Nefet comes after you and if not killed with FOD if he starts to fight you you will get kicked out of the quest. earlier I noticed if creatures hit the magic port to go to Ladios or the quest they would be found dead on the other end of the portal. So I decided to give this a try to see if he can be killed by something other than me. It took a couple of tries. I polymorphed Nefet from a distacne. Then as he started to come after me I hit him with the teleport away spell. Eventually i got him teleported outside of the castle and he was above the outside of the castle so I left and made my way to the level portal. I stood with the portal between me and Nefet. Eventually I was able to get him to step on the portal when trying to get to me and he disappeared. I had no way of knowing if he would be on the other side alive or dead. I used the eye to port to Ladios then went through the dungeon till I was at the level the port to the quest is at. As I approached the portal there was a corpse on the portal. I walked up and touched the portal and saw a message that Nefet dead body was on the port. Then the message saying the dungeon had collapsed around me and was directed to send the error in to be looked at. I only did this 1 time and do not know if this would have happened if I waited for the corpse to decay and disappear. I just assumed with the way Nefet died the game was in a error state and had to be ended. I just loaded my saved game and went back to normal play. I do not know if the collapse was from touching the corpse of Nefet, or from touching the portal as the quest level should be blocked due to attacking and causing Nefets death. Also at a later time I thought of using a helm of opposite alignment when killing Nefet to see if I could kill him with out upsetting my god and perhaps keep the use of the eye. I never got around to trying this as there is really nothing to gain from killing Nefet as he has no items unlike his apprentices. and you lose the ability to use the eye and can no longer port to parts of the dungeon. Still causing the dungeon to collapse and crash the game was kind of cool but totally a useless waste of time.

**Comment by dwntwn512** [comment]
![NH](https://user-images.githubusercontent.com/68635060/88151180-f9e9c900-cbc7-11ea-8fa4-868562747d76.jpg)
